app/models/user.rb

```diff
@@ -5,15 +5,39 @@ class User < ActiveRecord::Base
   has_secure_password
 
   has_many :queue_items, -> { order('position') }
-  has_many :reviews
+  has_many :reviews, -> { order('created_at DESC') }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.